packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -112,14 +110,12 @@ class="fixed inset-0 z-40 flex items-center min-h-screen p-4 overflow-y-auto tra
                             {{ $header }}
                         </div>
 
-                        @if ($isFullscreen)
-                            <x-filament-support::icon-button
-                                icon="heroicon-s-x"
-                                x-on:click="{{ filled($id) ? '$dispatch(\'' . $closeEventName . '\', { id: \'' . $id . '\' })' : 'isOpen = false' }}"
-                            >
-                                close
-                            </x-filament-support::icon-button>
-                        @endif
+                        <x-filament-support::icon-button
+                            icon="heroicon-s-x"
+                            x-on:click="{{ filled($id) ? '$dispatch(\'' . $closeEventName . '\', { id: \'' . $id . '\' })' : 'isOpen = false' }}"
+                        >
+                            close
```

Comment:
This serves no purpose?

Reply:
I need to do a translation for that but it's purpose is for screen readers. Should it just be an aria label on the button?

Reply:
As far as I can see, the icon button doesn't accept a slot?

Reply:
No worries. I'll just pass an aria-label. Should have a new commit in a little bit.

Reply:
Not sure how it's used in other places, but maybe we could pass a label for screen readers.

Reply:
We have a close button on #3847, merge that in

Reply:
i just passed a label to it, since that's built into the icon button.

Seems weird to me to use a component from notifications package in this context since they could have different appearance, but i'm happy to switch it out it that's what you want.

Reply:
In #3847 I added the feature to the support package. There is no dependency on the notifications package. If you remove your button and then merge that branch into this one, then you'll see it.

Reply:
@danharrin means he added a close button to the modal component, @awcodes.

Reply:
Ah. I misunderstood. I'll get to it shortly. Thanks.